### DISCUSSION TITLE
Use czkawka outside /home

### DISCUSSION THREAD

**Discussion by rolfbakker** [qarmin/czkawka#722] [Q&A] [answered]
Hi,
I saw czkawka featured in linux magazine, and was happy when reading, since this is a tool I'm looking for quite a while.
But I seem to do something wrong, or I miss an option.
I have my home on one drive, and then using symlinks some dirs on another drive:

```
rolf@argon:~> ls -l |grep ^l
lrwxrwxrwx   1 rolf users       16 Apr  1  2017 Documents -> /data/Documents/
lrwxrwxrwx   1 rolf users       11 Apr  2  2017 foto -> /data/foto/
lrwxrwxrwx   1 rolf users       10 Apr  2  2017 nextcloud -> /data/nextcloud/
```

But czkawka (or snap) does not seem to be able to handle this. I've tried using "snap connect czkawka:removable-media" but since this is not removable media, this does not work...
How can I use this tool to look for dupes in ~/foto vs ~/nextcloud?

Thanks,
Rolf

**Comment by qarmin** [comment]
This is expected behavior.
Enabling folder symlinks could introduce infinite loops and/or some performance degradation when checking if file is supported, so I don't think that at least for now is possible to add support for them.
The easiest workaournd is to point directly to `/data` folders in Czkawka but I understand that not always is this the best solution

**Comment by rolfbakker** [comment]
even worse: I already add /data/foto as folder (and not the symlink ~/foto, but the application errors:
Directories:  Provided folder path must exist, ignoring ⁨/data/foto⁩

**Comment by qarmin** [comment, marked as answer]
Can you check binaries from - https://github.com/qarmin/czkawka/releases/tag/4.1.0?
Snap are quite problematic in use

**Reply by rolfbakker** [reply]
yes, that works! First snap, and own compile did not work. But the precompiled binary did the trick!
Why hard, if it can be easy!
Thx!
